framework/core/src/Api/Controller/ClearCacheController.php

```diff
@@ -38,11 +45,24 @@ protected function delete(ServerRequestInterface $request)
     {
         RequestUtil::getActor($request)->assertAdmin();
 
-        $this->command->run(
+        $exitCode = $this->command->run(
             new ArrayInput([]),
             new NullOutput()
         );
 
+        if ($exitCode !== 0) {
+            throw new \Exception('Clearing cache failed. Try running `php flarum cache:clear` from the command line to see more info.');
```

Comment:
We can't throw a broad exception to the end user, it will just result in a server error without any meaningful message 🤔.

Let me push something for this real quick.